src/pretix/base/services/mail.py

```diff
@@ -332,3 +340,75 @@ def render_mail(template, context):
         tpl = get_template(template)
         body = tpl.render(context)
     return body
+
+
+def replace_images_with_cid_paths(body_html):
+    if body_html:
+        email = BeautifulSoup(body_html, "lxml")
+        image_counter = 1
+        cid_images = []
+        for image in email.findAll('img'):
+            cid_id = "image_%s" % (image_counter)
+            image_counter = image_counter + 1
+            original_image_src = image['src']
+
+            image['src'] = "cid:%s" % (cid_id)
+
+            cid_images.append({
+                'src': original_image_src,
+                'cid_id': cid_id
+            })
```

Comment:
Hmm, if I think about our sample email design, we use the same image multiple times, e.g. our separator bar. Could we use (at least internally in this method) a dict instead of a list here and build it in a way that the same ``src`` will only lead to one attachment, not multiple?